src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md

```diff
@@ -3,13 +3,46 @@ title: How to Create an HTML Button That Acts Like a Link
 ---
 ## How to Create an HTML Button That Acts Like a Link
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This guide will demonstrate "How to Create an HTML Button That Acts Like a Link".
+Programmers and Web Developers use buttons for several different ocassions. 
+Today you will create your very own social link button! 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+## Step 1 Button Tag
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+No not a HashTag, but a button tag. This is the first step and it is the step that defines to the computer
```

Comment:
Try to avoid saying "No not a HashTag, but a button tag" as this guide is mainly made for onlookers to be able to get quick information.